README.md

```diff
@@ -42,6 +42,27 @@ Several example notebooks are provided in the examples/ directory. Opening these
 
 Several examples can be found in the `examples/` directory (the [Fitting impedance spectra notebook](https://impedancepy.readthedocs.io/en/latest/examples/fitting_example.html) is a great place to start) and the documentation can be found at [impedancepy.readthedocs.io](https://impedancepy.readthedocs.io/en/latest/).
 
+## Citing impedance.py
+
+[![DOI](https://joss.theoj.org/papers/10.21105/joss.02349/status.svg)](https://doi.org/10.21105/joss.02349)
+
+If you use impedance.py in published work, please consider citing https://joss.theoj.org/papers/10.21105/joss.02349 as
+
+```bash
+@article{Murbach2020,
+  doi = {10.21105/joss.02349},
+  url = {https://doi.org/10.21105/joss.02349},
+  year = {2020},
+  publisher = {The Open Journal},
+  volume = {5},
+  number = {52},
+  pages = {2349},
+  author = {Matthew D. Murbach and Brian Gerwe and Neal Dawson-Elli and Lok-kun Tsui},
+  title = {impedance.py: A Python package for electrochemical impedance analysis},
```

Comment:
`for` is rendering in red as if it's a for loop

Reply:
Good eye 🎯 fixed in: https://github.com/ECSHackWeek/impedance.py/commit/9d8e2fdc7c92fcc4cd0528808a449875ecf0a861